netdata-installer.sh

```diff
@@ -1435,6 +1435,12 @@ should_install_ebpf() {
     return 1
   fi
 
+  if [ "$(uname -s)" != Linux ]; then
```

Comment:
not a bid deal but `"Linux"` looks better to me

Reply:
Fixed.